setup.py

```diff
@@ -8,7 +8,7 @@
     from distutils.core import setup
 
 install_requires = [
-    'requests >= 1.0.0',
+    'requests >= 1.2.3',
```

Comment:
# Reviewer Note

Confirmed that `v1.2.3` is the highest patch version of `v1.2.x` here: https://pypi.org/project/requests/#history